connectors/src/connectors/webcrawler/temporal/client.ts

```diff
@@ -16,15 +16,15 @@ import {
 } from "./workflows";
 
 export async function launchCrawlWebsiteWorkflow(
-  connectorId: ModelId
+  connectorId: ModelId,
+  priorityLevel: number | null
```

Comment:
priorityLevel could be replaced by priority and a typescript type of `"new_website" | "refresh_website" to be very explicit. Update the call sites accordingly.

Reply:
@lasryaric I used `number | null` to meet the `SyncConnector` signature